tests/micropython/viper_ptr16_store_boundary.py

```diff
@@ -1,5 +1,11 @@
 # Test boundary conditions for various architectures
 
+try:
+    int("0x10000000000000000", 16)
+except:
+    print("SKIP")  # No support for >64-bit integers
+    raise SystemExit
```

Comment:
Instead of this, could rename the test to end with `_intbig.py`.  Eg following `tests/micropython/viper_misc_intbig.py` and `tests/micropython/viper_const_intbig.py` etc.

Reply:
Oh of course! Thanks